src/openforms/config/models.py

```diff
@@ -229,6 +229,14 @@ class GlobalConfiguration(SingletonModel):
             "URL to the main website. Used for the 'back to municipality website' link."
         ),
     )
+    organization_name = models.CharField(
+        _("organization name"),
+        max_length=50,
```

Comment:
let's make this 100 to be future proof - longest municipality name at the moment seems to be around 30 chars, but if two of those fuse we're at 60+ characters.